### PR TITLE
Pull #3262: corrected alphabetical order of the checks in checks.xml

### PR DESCRIPTION
**Pull request by f05fk**
The "Checks" page lists the available checks in alphabetical order and even says that it does. Therefore I corrected the order of some checks, because they have not been in the right place.

### CODE REVIEW

**Comment by romani** [comment]
CI will fail.
we have strict rules on commit message please reword it to "Pull #3262: ........."

**Comment by f05fk** [comment]
It seems that this is a chicken-and-egg problem. When I want to have the commit comment of the form "Pull #3262: ..." then I first need the Pull request to get the number. But I cannot start a Pull request without a commit.

I will start-over from scratch tomorrow.

**Comment by romani** [comment]
Kind of., but not exactly the same.
You can change commmit message and push with force to the same your remote branch. That is all. All other will be done automatically.
